<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiteInfoPermissionItem</class>
 <widget class="QWidget" name="SiteInfoPermissionItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>66</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDefaultPermission">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAsk">
       <property name="text">
        <string>Ask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAllow">
       <property name="text">
        <string>Allow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioDeny">
       <property name="text">
        <string>Deny</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
